internal/v5/auth.go

```diff
@@ -457,3 +538,14 @@ func parseCredentials(req *http.Request) (username, password string, err error)
 	}
 	return tokens[0], tokens[1], nil
 }
+
+const condActiveTimeBefore = "active-time-before"
+
+// activeTimeBeforeCaveat returns a caveat that will be satisfied
+// before the given time; after the given time, it will only
+// be satisfied if the macaroon is being renewed.
+func activeTimeBeforeCaveat(t time.Time) checkers.Caveat {
+	return checkers.Caveat{
+		Condition: condActiveTimeBefore + " " + t.UTC().Format(time.RFC3339Nano),
```

Comment:
why not just RFC3339? i don't see much value in increased precision.. but it does mean the timestamp will take up more space in the cookie